pkg/controller/controller.go

```diff
@@ -58,39 +58,39 @@ func NewController(opts *options.BootstrapConfigs, bpfWorkloadObj *bpf.BpfKmeshW
 }
 
 func (c *Controller) Start(stopCh <-chan struct{}) error {
-	var secertManager *security.SecretManager
-	var err error
-	if c.mode == constants.WorkloadMode && c.enableSecretManager {
-		secertManager, err = security.NewSecretManager()
-		if err != nil {
-			return fmt.Errorf("secretManager create failed: %v", err)
-		}
-		go secertManager.Run(stopCh)
-	}
-
 	clientset, err := utils.GetK8sclient()
 	if err != nil {
 		return err
 	}
-	kmeshManageController, err := manage.NewKmeshManageController(clientset, secertManager)
-	if err != nil {
-		return fmt.Errorf("failed to start kmesh manage controller: %v", err)
-	}
-	go kmeshManageController.Run(stopCh)
-	log.Info("start kmesh manage controller successfully")
-
 	if c.enableByPass {
 		c := bypass.NewByPassController(clientset)
 		go c.Run(stopCh)
 		log.Info("start bypass controller successfully")
+		return nil
```

Comment:
we may still need to add some validation to prevent setting --mode with --enable-bypass simultaneously